config.go

```diff
@@ -405,6 +428,23 @@ func ParseConfig(fn string) (Configuration, error) {
 				config.S3ObjectPrefix = &objectPrefix
 			}
 
+			val, ok = input.Get("s3", "verbose_key")
```

Comment:
They may or may not want this verbose key option. It's really just for us and the way we separate data in s3. Be prepared to pull it out.